### PR TITLE
Target Remote.Workspaces to .NET Core 3.1

### PR DESCRIPTION
**Pull request by genlu**
@tmat @sharwell

### CODE REVIEW

**Comment by tmat** [comment]
I wonder if there is much benefit right now to target `net472` and `netcoreapp3.1` vs. `netstandard2.0`?

**Comment by genlu** [comment]
Not sure. I'm trying to be consistent with what @sharwell is doing.

**Comment by sharwell** [comment]
@tmat netstandard2.0 is not a supported scenario for C# 8 or nullable reference types

**Comment by tmat** [comment]
OK, as long as we know what the reasons are. Just keep in mind that there might be a trade-off: if we make all projects multi-targeted we make Roslyn.sln much slower to work in.

File: src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.Remote</RootNamespace>
-    <TargetFramework>net472</TargetFramework>
+    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
```
Comment (sharwell) [inline src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj:7]: 📝 `netcoreapp3.1` should appear first in the list for consistency

**Pull request merged**